scripts/upload-static-assets.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Check for force env var or production environment
+if [[ "$FORCE_ASSET_CDN" != "1" ]] && [[ "$VERCEL_ENV" != "production" ]]; then
```

Comment:
This script will need to finish before Vercel prod build for studio, right? When restoring from cache, the Vercel build can take ~1 minute.

Should we bother syncronizing them?

Reply:
I think the deployment waits on the upload to finish before it switches things over. Check out the build logs here: https://vercel.com/supabase/studio-staging/9dJWq5UAFM6Eg8Yp8xTNvCq791j4

Is that what you mean by synchronising them?

Reply:
Ah, the upload happens by a Vercel worker, I hallucinated that it's by a Github worker, sorry.